tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -281,6 +299,49 @@ private void handleRunTests(List<String> toolNames, List<String> sources) {
         }
     }
 
+
+    private void setUpGa4Ghv20Api() {
+        this.tooltesterConfig = new TooltesterConfig();
+        io.dockstore.openapi.client.ApiClient defaultApiClient = io.dockstore.openapi.client.Configuration.getDefaultApiClient();
+        defaultApiClient.setBasePath(this.tooltesterConfig.getServerUrl());
+        setGa4Ghv20Api(new Ga4Ghv20Api(defaultApiClient));
+    }
+
+    private void runToolTesterOnWorkflows() throws InterruptedException {
+        List<WorkflowRunner> workflowsToRun = new ArrayList<>();
+        setUpGa4Ghv20Api();
+
+        // These are just left here for testing purposes, until I find the API commands to get them
+        workflowsToRun.add(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "Dockstore.json"));
+        workflowsToRun.add(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "/agc-examples/fastq/input.json", getGa4Ghv20Api()));
```

Comment:
This is an example of running a workflow that downloads the test parameter file from the `Ga4Ghv20Api`.

Reply:
I will move the location of the `workflowsToRun`, probably to their own class like what is down for the BlackList?

https://github.com/dockstore/dockstore-support/blob/d234c0087785cfa9e3733e3f5c0146289cf43f47/tooltester/src/main/java/io/dockstore/tooltester/blacklist/BlackList.java

Reply:
A new class was created to hold these called `WorkflowList`.